pyproject.toml

```diff
@@ -57,6 +67,7 @@ channels = ['conda-forge']
 
 [tool.isort]
 # settings for compatibility between ``isort`` and ``black`` formatting
+known_third_party = ["geoh5py", "geoapps_utils"]
```

Comment:
probably not necessary: pylint had an issue but not isort. I will revert that line